src/containers/Amendments/test/mockAmendments.json

```diff
@@ -0,0 +1,137 @@
+
+{
```

Comment:
Is there a reason the VHS api breaks these out into two arrays?

Reply:
Fixed the two arrays